middleware/file/unittest/source/resource/utility.cpp

```diff
@@ -0,0 +1,71 @@
+//! 
+//! Copyright (c) 2019, The casual project
+//!
+//! This software is licensed under the MIT license, https://opensource.org/licenses/MIT
+//!
+
+#include "file/resource/unittest/utility.h"
+
+#include "file/manager/admin/services.h"
+
+#include "file/message.h"
+#include "file/instance.h"
+
+#include "serviceframework/service/protocol/call.h"
+
+namespace casual
+{
+   namespace file::resource::unittest
+   {
+      manager::admin::model::State state()
+      {
+         serviceframework::service::protocol::binary::Call call;
+         auto reply = call( manager::admin::service::name::state);
+
+         decltype( state()) result;
+         reply >> CASUAL_NAMED_VALUE( result);
+         return result;
+      }
+
+
+      namespace acquire
+      {
+         namespace local
+         {
+            namespace device
+            {
+               namespace
```

Comment:
Have the unnamed ns directly under _local_.

Reply:
What is the rational for that ?

Reply:
* consistency 
* When someone later ads stuff directly under local it might not get internal linkage
* IDE:s gets confused. Maybe this is not an issue any more, but it certainly was a problem.